packages/spectral/src/generators/componentManifest/templates/actions/action.ts.ejs

```diff
@@ -12,9 +13,12 @@ export interface <%= action.typeInterface %>Values {
  */
 export const <%= action.import %> = {
   key: "<%= action.key %>",
-  perform: <TReturn>(
-    _values: <%= action.typeInterface %>Values
-  ): Promise<TReturn> => Promise.resolve<TReturn>({} as TReturn),
+  perform: async <TReturn>(
+    values: <%= action.typeInterface %>Values
+  ): Promise<TReturn> => {
+    const context = requireContext();
+    return await context.components.<%= action.component %>.<%= action.key %>({ ...values }) as TReturn;
```

Comment:
Just to make sure I'm thinking about this correctly - we don't need to worry about existing _published_ component manifests which don't have `action.component` defined following this code path because this will only be generated for net-new components and only exposed as a callable function (at least reasonably discoverable) for inline generated manifest. Is that correct?